awscli/examples/rum/list-app-monitors.rst

```diff
@@ -0,0 +1,38 @@
+**To return a list of the Amazon CloudWatch RUM app monitors in the account**
+
+The following ``list-app-monitors`` returns a list of the Amazon CloudWatch RUM app monitors in the account. ::
+
+	aws rum list-app-monitors
+
+Output::
+
+	{
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general